textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/KeywordPredicate.java

```diff
@@ -135,7 +135,7 @@ public Analyzer getAnalyzer(){
     }
 
     public DataReaderPredicate getDataReaderPredicate() {
-        DataReaderPredicate dataReaderPredicate = new DataReaderPredicate(this.dataStore, this.luceneQuery);
+        DataReaderPredicate dataReaderPredicate = new DataReaderPredicate(this.dataStore, this.luceneQuery, this.query, this.analyzer,this.attributeList);
```

Comment:
Since we want the Lucene dependency more clear in variable names, whenever possible, change `analyzer` to `luceneAnalyzer`.